index.html

```diff
@@ -184,6 +184,106 @@ <h2 class="project-title">Tonic</h2>
           </p>
         </div>
         <div class="project-tag">
+          <ul class="project-skills flex">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+        </div>
+        <button class="see-project-btn" type="button">See Project</button>
+      </div>
+    </div>
+  </section>
+  <section id="about" class="about">
+    <div class="about-container flex column">
+      <div class="abt-me flex column">
+        <h2 class="header-text">About <br>Myself</h2>
+        <p class="para-text">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <div class="self-container">
+          <p>Let's Connect</p>
+          <div class="list-nav">
+            <ul class="list-items skill-items">
```

Comment:
- The LinkedIn icon is one image. Kindly download it in this way.
![image](https://user-images.githubusercontent.com/61361037/219857048-cfcc4c57-c2c7-4792-a439-d7db3c8c051e.png)

- The size of the font icons in Figma design is around `20px`. The width and height change as per the shape of the icon. I am sharing the sizes comparison: Kindly apply that size to get near to the design.

Reply:
@RaoAkif on the screenshot, it's a github icon... when i'm trying to download linkedin icon, it downloads it seperatly... how can i download it in correct way please?

Reply:
@RaoAkif i found, it's okay